modules/perception/obstacle/onboard/camera_process_subnode.h

```diff
@@ -90,6 +90,8 @@ class CameraProcessSubnode : public Subnode {
   CameraSharedData* cam_shared_data_;
 
   // Calibration
+  int32_t image_height_ = 1080;
```

Comment:
consider change this one to common or gflags maybe? I saw these two in different image related files.

Reply:
Will read it from Calibration service, which is not used now